src/spinner.tsx

```diff
@@ -30,23 +31,20 @@ const Spinner = ({
     const {texts, platformOverrides, t} = useTheme();
     const isInverse = useIsInverseOrMediaVariant();
     color = color || (isInverse ? vars.colors.controlActivatedInverse : vars.colors.controlActivated);
-    const spinnerId = React.useId();
-    const title = texts.loading || t(tokens.loading);
+    const label = texts.loading || t(tokens.loading);
     const content =
         getPlatform(platformOverrides) === 'ios' ? (
             <svg
-                aria-labelledby={spinnerId}
                 className={styles.spinnerIos}
                 height={size}
                 style={{...style}}
                 role="progressbar"
                 aria-live="polite"
-                aria-busy
```

Comment:
this doesn't make sense here and makes voiceover fail reading the progressbar. `aria-busy` is to be used in the aria that will load the content when the loading finished, not in the spinner/progressbar